templates/popping-our-problems.html

```diff
@@ -9,7 +9,8 @@
     </div>
     <div class="panel s0">
         <p>
-             Let's look at how Mia used the POP technique.
+            Now we'll break down the POP technique, see how Mia worked through the three steps. You'll watch three clips.
```

Comment:
Maybe a transition word instead of the comma? Something like: "Now we'll break down the POP technique to see how Mia worked through the three steps. "